client/status/viewmodel/render.go

```diff
@@ -493,15 +495,17 @@ func renderPrunerReport(t *stringbuilder.B, r *pruner.Report, fsfilter FilterFun
 	}
 
 	// global progress bar
-	progress := int(math.Round(80 * float64(completedDestroyCount) / float64(totalDestroyCount)))
-	t.Write("Progress: ")
-	t.Write("[")
-	t.Write(stringbuilder.Times("=", progress))
-	t.Write(">")
-	t.Write(stringbuilder.Times("-", 80-progress))
-	t.Write("]")
-	t.Printf(" %d/%d snapshots", completedDestroyCount, totalDestroyCount)
-	t.Newline()
+	if state != pruner.Done {
```

Comment:
Similarly, for pruner, terminal states are  `PlanErr`, `ExecErr` and `Done`

Learn more by checking the places where `doOneAttempt` sets the state and returns

https://github.com/zrepl/zrepl/blob/1f0f2f856966713c06b76943d9de0c560f820557/daemon/pruner/pruner.go#L352

Reply:
Same as `AttemptState`.